package.json

```diff
@@ -728,6 +728,41 @@
           "markdownDescription": "Allows colors for the given file name patterns. See [here](command:xml.open.docs?%5B%7B%22page%22%3A%22Features/XMLColorsFeatures%22%2C%22section%22%3A%22xmlcolorsfeatures%22%7D%5D) for more information.",
           "scope": "window"
         },
+        "xml.files": {
```

Comment:
xml.completion.pathsupport ?

Reply:
I would like to keep `xml.files` because today it supports only completion,but my idea is to support too documentLink and validation.

Reply:
fixed